mlflow/server/js/src/components/ExperimentView.css

```diff
@@ -113,13 +114,22 @@
 }
 
 .ExperimentView-search-controls .filter-label {
-  width: 92px;
+  width: 84px; /* COMMON_FILTER_LABEL_WIDTH */
```

Comment:
Style fixes are on spot for this PR because we are going to implement more concise UI for search/filtering soon.